ZUGFeRD/Party.cs

```diff
@@ -63,5 +63,15 @@ public class Party
       /// </summary>
       public string Street { get; set; }
       public GlobalID GlobalID { get; set; }
-   }
+
+      /// <summary>
+      /// Address line 3
+      /// </summary>
+      public string AddressLine3 { get; set; }
+
+      /// <summary>
+      /// Country subdivision
+      /// </summary>
+      public string CountrySubdivisionName { get; set; }

```

Comment:
same here: Could you add a text here for the purpose of thies field and that it is purely optional